### ISSUE TITLE
Custom dataset and splits

### ISSUE THREAD

**Issue by i-makivskiy** [nianticlabs/monodepth2#187]
Hi!
Thanks for you amazing job, it's really powerful and interesting thing, but i have few questions:

1. Splits are uses for determining which side of camera using? Two sides are necessary for training or i can use only single camera?
2. There is any restrictions to use it inside building (trained on custom data)?
3. Which loss value are normal for first epoch? Right now i have 0.03, and it's look's like not good.

With best regards. Ihor.

**Comment by mrharicot** [comment]
Hi,

1. You can use a single camera, we indicate which side in the .txt files so that we can handle stereo videos.
2. If it was trained on kitti it will most likely struggle to produce good results in indoors data. Training on indoors data is also quite challenging.
3. Hard to say, it depends on the dataset, but the loss should decrease fairly gradually.

**Comment by i-makivskiy** [comment]
Thanks for your answers.

We trained on our data, and result is unexpected:
![image](https://user-images.githubusercontent.com/47322074/82357704-22114c00-9a0e-11ea-8f9b-10b102159a7d.png)

Our train data is few videos from few different locations(homes/office), and some videos very unstable, It's can produce troubles?

And about intrinsics matrix - if we use videos from few different phones - we need use few different matrix for every camera? 

And another one question:
In mono_dataset.py, get_item function we use frame_idxs which taken from params or default values [0, 1, -1], i correct to understand, what it's taken randomly some frame, or -1/+1 frame?

Sorry, if my questions is stupid, and thanks for your patience!

With best regards. Ihor.

**Comment by mrharicot** [comment]
Hi,
monodepth2 hasn't really been tested on indoor data, it's a significantly harder problem so I am not surprised it is struggling to train.
Yes if the video comes from multiple phones you will need to adapt the dataloader to return a different camera intrinsics for each sample.
As for the `[0, 1, -1]` they correspond to the indices of images to load relative to the index from the txt file.

**Comment by Rishav-Paramhans** [comment]
Hie. First of all thank you for the great work. I am training the model on KITTI Raw data for monocular depth estimation. I am getting depths map like the one I have attached here. I am not really able to find out the problem why my depth maps come out like this.. 
![download](https://user-images.githubusercontent.com/65668108/194886798-6b659faa-4b9f-4384-a2dd-db1a5fe967d4.png)

**Comment by lhx0416** [comment]
> Hie. First of all thank you for the great work. I am training the model on KITTI Raw data for monocular depth estimation. I am getting depths map like the one I have attached here. I am not really able to find out the problem why my depth maps come out like this.. ![download](https://user-images.githubusercontent.com/65668108/194886798-6b659faa-4b9f-4384-a2dd-db1a5fe967d4.png)

Hi，i meet the same question. have you solved it?
